SEED-platform/seed
# refactor(building_sync): handle custom exception

#### Any background context you want to provide?
Currently, when updating a property with a buildingsync file, if there's a parsing error the server just 500's without explaining why.
#### What's this PR do?
Adds custom exception type for basic parsing of bsync, and returns it as an error message if caught
#### How should this be manually tested?
Upload a property. On details page, click "update with buildingsync" and choose a _bad_ xml file (e.g. some greenbutton data). It should alert you about what was wrong. Then try uploading a valid bsync file and it should be ok.
